core/__tests__/models/app/modelCache.ts

```diff
@@ -99,10 +94,15 @@ describe("models/appsCache", () => {
       expect(AppsCache.expires).toEqual(0);
     });
 
+    beforeEach(async () => {
+      await helper.sleep(1000); // wait for any pub/sub to complete
+      AppsCache.expires = 0;
+      await AppsCache.findAllWithCache();
+    });
```

Comment:
This is the main change in all the tests.  The previous sleeps were both not long enough in some cases when the host is overloaded (like on CI) and not applied before the tests that need it.